react/features/base/react/components/web/JitsiSectionList.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import React, { Component } from 'react';
+import { JitsiSectionListSection } from './index';
+
+type Props = {
+
+    /**
+     * Sections to be rendered in the following format:
+     *
+     * [
+     *   {
+     *     title: string,               <- section title
+     *     key: string,                 <- unique key for the section
+     *     data: [                      <- Array of items in the section
+     *       {
+     *         colorBase: string,       <- the color base of the avatar
+     *         title: string,           <- item title
+     *         url: string,             <- item url
+     *         lines: Array<string>     <- additional lines to be rendered
+     *       }
+     *     ]
+     *   }
+     * ]
+     */
+    sections: Array<Object>,
+
+    /**
+     * onItemClick is a function that defines what happens when
+     * an item in the section list is clicked
+     */
+    onItemClick: Function
+};
+
+/**
+ * Implements a React/Web {@link Component} for displaying a list with
+ * sections similar to React Native's {@code SectionList} in order to
+ * faciliate cross-platform source code.
+ *
+ * @extends Component
+ */
+export default class JitsiSectionList extends Component<Props> {
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+
+
+    /**
+     * Renders the content of this component.
+     *
+     * @returns {React.ReactNode}
+     */
+    render() {
+        return (
+            /* eslint-disable no-extra-parens */
```

Comment:
Why do you need this? 

Even if you need it it's better to use eslint-disable-next-line or eslint-disable-line. If you need to use eslint-disable, please enable it after that when it is not needed anymore.